src/main/java/org/jboss/forge/arquillian/api/AlgeronSetupFacet.java

```diff
@@ -0,0 +1,78 @@
+package org.jboss.forge.arquillian.api;
+
+import org.jboss.forge.addon.dependencies.Coordinate;
+import org.jboss.forge.addon.dependencies.builder.DependencyBuilder;
+import org.jboss.forge.addon.facets.constraints.FacetConstraint;
+import org.jboss.forge.addon.projects.facets.DependencyFacet;
+import org.jboss.forge.addon.projects.facets.MetadataFacet;
+
+@FacetConstraint(TestFrameworkFacet.class)
```

Comment:
If that doesn't work, check the `isInstalled` method for the TestFrameworkFacet implementations.